jest/node-exports-resolver.js

```diff
@@ -54,7 +54,7 @@ function getPackageJson (packageName) {
     return require(`${packageName}/package.json`)
```

Comment:
In summary I had to change the temporary jest resolver to fix some issues with resolving `packageName=@bugsnag/core, submoduleName=.` by removing the ` && submoduleName !== '.'` condition. But then needed to return null instead of erroring for node modules that resolve with e.g. `packageName=https, submoduleName=.`

Hopefully we'll be able to bump jest later and we'll be able to drop this